WordPress/Classes/Services/Store.swift

```diff
@@ -48,33 +48,31 @@ enum StoreCoordinatorError: ErrorType {
 ///   purchased product, as well as a localized error message under `NSUnderlyingErrorKey`.
 class StoreCoordinator<S: Store> {
     private let store: S
+    private let keyValueStorage: KeyValueStorage
```

Comment:
I'd rename this, not sure to what. Naming it `keyValueStorage` feels like naming something `dictionary`. Maybe `settings` or `database`.